person.rb

```diff
@@ -24,5 +26,9 @@ def validate_name
     @name = @corrector.correct_name(@name)
   end
 
+  def add_rentals(date, book)
+    Rental.new(date, self, book)
+  end
+
```

Comment:
- [ ] You have missed adding `rentals` as the `attr_accessor` field. Please add it.